djangocms_page_meta/utils.py

```diff
@@ -93,7 +93,7 @@ def get_page_meta(page, language):
             "twitter_site": meta_settings.TWITTER_SITE,
             "twitter_author": meta_settings.TWITTER_AUTHOR,
             "schemaorg_type": meta_settings.SCHEMAORG_TYPE,
-            "schemaorg_datePublished": page.publication_date.isoformat() if page.publication_date else None,
+            "schemaorg_datePublished": page.creation_date.isoformat() if page.creation_date else None,
```

Comment:
I think if versioning is installed this would need to use the published date, it will default to creation date if versioning is not installed. Content expiry would also then handle publication end date.